playwright/_impl/_transport.py

```diff
@@ -154,20 +153,20 @@ def __init__(
         super().__init__(loop)
         Transport.__init__(self, loop)
 
-        self._stopped = False
         self.ws_endpoint = ws_endpoint
         self.headers = headers
         self.slow_mo = slow_mo
 
     def request_stop(self) -> None:
-        self._stopped = True
+        self._stopped.set()
```

Comment:
We should not notify those that wait for stop here, we do that when we actually stopped. Is there a bug for this? What are we fixting?

Reply:
We do wait for the actual connection to close https://github.com/microsoft/playwright-python/blob/ef07845d4433467798dcabe29daefa737f8518f9/playwright/_impl/_transport.py#L170

Reply:
Can you show a failing test with the current change which was passing previously to show your point better ?, fyi: this change does not break anything

Reply:
@mxschmitt Your opinion here would be appreciated!